gamemodes/prop_hunt/gamemode/lang/english.lua

```diff
@@ -1,4 +1,4 @@
-LANG = {}
+local LANG = {}
```

Comment:
90% sure LANG needs to be a global file otherwise it can't be referenced outside of this script.